ext/fg/js/popup.js

```diff
@@ -287,8 +315,11 @@ class Popup {
             }, timeout);
 
             chrome.runtime.onMessage.addListener(onMessage);
-            frame.addEventListener('load', () => containerLoadedResolve());
-            frame.addEventListener('error', () => containerLoadedReject(new Error('Failed to load container')));
```

Comment:
Why was `error` handler removed here?

Reply:
I don't think "error" is a real event for frames. You get load events even for invalid content.